### DISCUSSION TITLE
Which one should I use for setup on ARM vps: Stable or Canary?

### DISCUSSION THREAD

**Discussion by ewallz** [easypanel-io/community#79] [Q&A] [answered]
Hi, i already got one installed on x86 vps using the stable installation. And i want to install another panel on an arm64 vps. Should i use the standard stable install or use the canary build?
https://easypanel.io/docs

**Comment by deiucanta** [comment, marked as answer]
You should use stable as well. The caveat with ARM is that some templates don't have Docker images that are ARM-compatible. Other than this, everything should work just fine.

**Reply by ewallz** [reply]
well noted on the images availability for the templates. so if the templates support arm, easypanel will automatically select the arm version right?

**Reply by deiucanta** [reply]
correct
